packages/fx-core/resource/package.nls.json

```diff
@@ -47,6 +47,7 @@
   "core.migrationToArmAndMultiEnv.ErrorMessage": "Upgrade Failed. Please check the error message in Output window to troubleshoot and fix. You can also click Learn More for the FAQ and how to upgrade manually.",
   "core.migrationToArmAndMultiEnv.PreCheckErrorMessage": "Upgrade Failed. '%s' doesn't exist or is not in json format. Please fix it and try again by running command (Teams: Upgrade project).\nYou can also click Learn More for the FAQ.",
   "core.consolidateLocalRemote.Message": "Your project will be upgraded to work with the latest version of Teams Toolkit. The upgrade will only backup and change the local config and manifest template file without changing your custom code. If you want to skip auto-upgrade, please install older version of Teams Toolkit (<=3.7.0)",
+  "core.consolidateLocalRemote.DifferentManifest": "Since Teams Toolkit will use manifest.remote.template.json as a single manifest template file after the upgrade, if you have customized the manifest.local.template.json you will need to place your changes in config.local.json and pass it to the template.",
```

Comment:
`if you have customized the manifest.local.template.json you will need to place your changes in config.local.json and pass it to the template.` This might not be very accurate, probably should say: `if you have customized the manifest.local.template.json, you will need to update the manifest.template.json to include your change. `

Reply:
Thanks for the advice, ```unify-config-markdown``` and ```readme``` doc have been updated..